app/api/dao/admin.py

```diff
@@ -1,13 +1,15 @@
-from app import messages
+from app import messages, messenger
 from app.database.models.user import UserModel
+from app.messages_enums import Message
 from app.utils.decorator_utils import email_verification_required
 
+
 class AdminDAO:
     """Data Access Object for Admin functionalities."""
 
     @staticmethod
     @email_verification_required
-    def assign_new_user(user_id, data):
+    def assign_new_user(user_id, data, lang="en"):
```

Comment:
Not sure, but there could be a way to define the `lang` parameter definition one time for every endpoint. Not sure if that's possible though 🤔 Since it can become cumbersome if we have to add this to every API function. Also because some of this functions are generated at the DAO level, some of these functions might also have to receive the `lang` argument.

Reply:
Yeah, I was also thinking about it but decided to wait for response before proceeding with more complicated stuff. Anyway, I'm gonna find some solution which fits nicely and ping you back for a review:)